controller/internal/supervisor/iptablesctrl/iptables.go

```diff
@@ -209,6 +212,10 @@ func (i *iptables) Run(ctx context.Context) error {
 		return fmt.Errorf("failed to update synack networks: %s", err)
 	}
 
+	if err := i.impl.Commit(); err != nil {
```

Comment:
why is the additional commit necessary now?

Reply:
Well this is interesting. The commit should have  been there always. We use to rely on iptables version. Old versions don't support save and restore and new iptables do. For new iptables commit is required for any effect to take place, Since all the iptables related changes are only in memory. Since we packaged a new iptables binary this was required.

Although this was a bug, it didnt affect any worflows because this was at the time of initialization. When the PU comes up the code uses iptables.commit() always. But for some reason the commit got missed in the init phase.

Reply:
interesting :)